sources/packages/api/src/database/entities/sfas-part-time-application.model.ts

```diff
@@ -0,0 +1,68 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+import { BaseModel } from ".";
+import { TableNames } from "../constant";
+import { dateOnlyTransformer } from "../transformers/date-only.transformer";
+
+/**
+ * Data related to a Student Application on SFAS Part Time.
+ */
+@Entity({ name: TableNames.SFASPartTimeApplications })
+export class SFASPartTimeApplications extends BaseModel {
+  /**
+   * The unique key/number used in SFAS to identify this application (Sail_extract_data.sail_application_no).
+   */
```

Comment:
Please be consistent on the 'Sail_extract_data' in the comments of the model

Reply:
commented above